src/features/team-load-channel.js

```diff
@@ -0,0 +1,156 @@
+                }
+            })
+            .fail(function(d) {
+                debug.log("check if "+uName+" is subbed to "+chan+" failed");
+            });
+        }
+    }
+
+    var addStoof = function(val) {
+        var jsnTeam = vars.jsnTeam;
+        
+        for(var i=0; i<jsnTeam.length; i++) {
+            if(jsnTeam[i].channel.name == chan) {
+                $("#channel_url").val("http://www.twitch.tv/"+chan);
+                $("#live_embed").val('<object type="application/x-shockwave-flash" height="378" width="620" id="live_embed_player_flash" data="http://www.twitch.tv/widgets/live_embed_player.swf?channel='+chan+'" bgcolor="#000000"><param name="allowFullScreen" value="true" /><param name="allowScriptAccess" value="always" /><param name="allowNetworking" value="all" /><param name="movie" value="http://www.twitch.tv/widgets/live_embed_player.swf" /><param name="flashvars" value="hostname=www.twitch.tv&channel='+chan+'&auto_play=true&start_volume=25" /></object>');
+
+                $("#facebook_like_button").empty();
+                var faceBookFrame = $("<iframe>", {"id":"facebook_like_iframe", "frameborder":"0", "allowtransparency":"true", "src":"http://www.facebook.com/plugins/like.php?href=http://www.twitch.tv/"+chan+"&layout=button_count&show-faces=false&share=false&action=like&width=85&height=21&colorscheme=light"});
+                $("#facebook_like_button").append(faceBookFrame);
+
+                $("#twitter_share_button").empty();
+                var tweetBtn = $("<a>", {"class":"twitter-share-button", "href":"https://twitter.com/share", "data-url":"http://www.twitch.tv/"+chan, "data-text":jsnTeam[i].channel.display_name+" is playing "+jsnTeam[i].channel.meta_game+" at:"}),
+                    tweetJS = $("<script>", {"src":"http://platform.twitter.com/widgets.js", "type":"text/javascript"});
+                tweetBtn.text("Tweet");
+                $("#twitter_share_button").append(tweetBtn, tweetJS);
+
+                var chanLinkSpan = $("<strong>"),
+                    chanLink = $("<a>", {"id":"live_channel_name", "href":"/"+chan});
+                chanLink.text(jsnTeam[i].channel.display_name);
+                chanLinkSpan.append(chanLink);
+
+                var viewerCount = $("<span>", {"id":"channel_viewer_count", "class":"stat"});
+                viewerCount.text(jsnTeam[i].channel.current_viewers);
+
+                var viewsCount = $("<span>", {"id":"views_count", "class":"stat"});
+                viewsCount.text(jsnTeam[i].channel.total_views+" ");
+
+                var followersCount = $("<span>", {"id":"followers_count", "class":"stat"});
+                followersCount.text(jsnTeam[i].channel.followers_count);
+
+                var descHolder = $("<div>", {"id":"description"});
+                descHolder.html("<b>Channel Description:</b><br>"+jsnTeam[i].channel.description+"<br><br><b>Broadcast Title:</b><br>"+jsnTeam[i].channel.title);
```

Comment:
HTML should be placed in jade templates and imported: https://github.com/night/BetterTTV/blob/master/src/templates/setting-switch.jade